custom_components/midea_ac_lan/__init__.py

```diff
@@ -76,7 +76,7 @@ async def update_listener(hass: HomeAssistant, config_entry: ConfigEntry) -> Non
             dev.set_refresh_interval(refresh_interval)
 
 
-async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:  # noqa: ARG001
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> None:  # noqa: ARG001
```

Comment:
ERROR:
`2024-09-01 01:08:11.363 ERROR (MainThread) [homeassistant.setup] Setup failed for custom integration 'midea_ac_lan': Integration 'midea_ac_lan' did not return boolean if setup was successful. Disabling component.`